frontend/src/metabase/ui/components/inputs/Select/Select.styled.tsx

```diff
@@ -0,0 +1,76 @@
+import { rem } from "@mantine/core";
+import type { MantineThemeOverride } from "@mantine/core";
+import { SelectItem } from "./SelectItem";
+
+export const getSelectOverrides = (): MantineThemeOverride["components"] => ({
+  Select: {
+    defaultProps: {
+      withinPortal: true,
+      itemComponent: SelectItem,
+      maxDropdownHeight: 512,
+    },
+    styles: theme => ({
+      wrapper: {
+        marginTop: theme.spacing.xs,
+      },
+      rightSection: {
+        svg: {
+          color: theme.colors.text[2],
+          width: rem(16),
+          height: rem(16),
+
+          "&[data-chevron] path": {
+            d: 'path("M 1.38 4.19 a 0.7 0.7 90 0 1 0.99 0 L 7.5 9.32 l 5.13 -5.13 a 0.7 0.7 90 1 1 0.99 0.99 l -5.63 5.63 a 0.7 0.7 90 0 1 -0.99 0 l -5.63 -5.63 a 0.7 0.7 90 0 1 0 -0.99 z")',
```

Comment:
Select doesn't allow to override icons otherwise

Reply:
Have you seen https://mantine.dev/core/select/#right-section ? if I understood it correctly it does what we need, but we'd have to handle `clearable` with some custom code